### PR TITLE
Adding expect_column_values_to_be_valid_urls expectation

### PR DESCRIPTION
**Pull request by ckathleen**
Expectation validates whether the column contains valid URL.

### CODE REVIEW

File: contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_valid_urls.py [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    _registered_metrics,
+    _registered_renderers,
+)
+from great_expectations.expectations.util import render_evaluation_parameter_string
+from great_expectations.render.renderer.renderer import renderer
+from great_expectations.render.types import RenderedStringTemplateContent
+from great_expectations.render.util import num_to_str, substitute_none_for_missing
+from great_expectations.validator.validator import Validator
+
+# This method compares a string to a url validation regex using Django's URLValidator class
```
Comment (Shinnnyshinshin) [inline contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_valid_urls.py]:
```suggestion


# This method compares a string to a url validation regex using Django's URLValidator class
```

Comment (Shinnnyshinshin) [reply contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_valid_urls.py]: This is one change requested by our linter

**Comment by Shinnnyshinshin** [comment]
Hi @ckathleen thank you again for your work with this. The code looks absolutely wonderful, but the one blocker right now is the CLA. If you'd able to get that signed, then we should be able to move this PR forward :)

File: contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_valid_urls.py [hunk 2]
```diff
@@ -0,0 +1,238 @@
...
+        "maturity": "experimental",
+        "hackathon"  # "experimental", "beta", or "production"
+        "tags": [  # Tags for this Expectation in the gallery
+            #         "experimental"
+        ],
+        "contributors": [  # Github handles for all contributors to this Expectation
+            "ckathleen"
+        ],
+        "package": "experimental_expectations",
+    }
```
Comment (Shinnnyshinshin) [inline contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_valid_urls.py]:
```suggestion
    library_metadata = {
        "maturity": "experimental",
        "tags": ["experimental", "hackathon"],
        "contributors": [  # Github handles for all contributors to this Expectation
            "@ckathleen"
        ],
        "package": "experimental_expectations",
    }
```

File: contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_valid_urls.py (hunk 1 above)
Comment (Shinnnyshinshin) [inline contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_valid_urls.py]: I had a question regarding the `Django's URLValidator class` comment.. is this class still used in this version of the PR?

**Comment by Shinnnyshinshin** [comment]
Seeing if this comment will run @cla-bot check again

**Comment by Shinnnyshinshin** [comment]
Doesn't look like that worked.. I'll ping @kyle to see if it's something on our end

**Comment by kyleaton** [comment]
@cla-bot check

**Comment by kyleaton** [comment]
@cla-bot check

File: contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_valid_urls.py [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+    _registered_renderers,
+)
+from great_expectations.expectations.util import render_evaluation_parameter_string
+from great_expectations.render.renderer.renderer import renderer
+from great_expectations.render.types import RenderedStringTemplateContent
+from great_expectations.render.util import num_to_str, substitute_none_for_missing
+from great_expectations.validator.validator import Validator
+
+
+# This method compares a string to a url validation regex using Django's URLValidator class
```
Comment (Shinnnyshinshin) [inline contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_valid_urls.py]:
```suggestion
# This method compares a string to a url validation regex
```

File: contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_valid_urls.py [hunk 4]
```diff
@@ -0,0 +1,236 @@
...
+# This method defines the business logic for evaluating your metric when using a SparkDFExecutionEngine
+#     @column_condition_partial(engine=SparkDFExecutionEngine)
+#     def _spark(cls, column, **kwargs):
+#         return column.isin([3])
+
+
+# This class defines the Expectation itself
+# The main business logic for calculation lives here.
+class ExpectColumnValuesToBeValidUrls(ColumnMapExpectation):
+    """Expect the column to be a valid url. Uses UrlValidator class in Django."""
```
Comment (Shinnnyshinshin) [inline contrib/experimental/great_expectations_experimental/expectations/expect_column_values_to_be_valid_urls.py]:
```suggestion
class ExpectColumnValuesToBeValidUrls(ColumnMapExpectation):
    """Expect the column to be a valid url.  Maps row values to regex to check if value is a valid url."""
```

**Review by Shinnnyshinshin — APPROVED** [review]

**Comment by Shinnnyshinshin** [comment]
Hi @ckathleen I did some minor clean up, and have approved the PR.  Thank you very much for your contribution here, and I hope we get to work again in the future :) have a great one!

**Comment by ckathleen** [comment]
Amazing!! Thank you!!

**Pull request merged**